packages/server/graphql/mutations/helpers/canAccessAISummary.ts

```diff
@@ -2,15 +2,26 @@ import {Threshold} from 'parabol-client/types/constEnums'
 import {Team} from '../../../postgres/queries/getTeamsByIds'
 import {DataLoaderWorker} from '../../graphql'
 
-const canAccessAISummary = async (
+// Returns whether AI summaries are allowed according to the user's + org's feature flags.
+export const isAISummaryAllowed = async (
   team: Team,
   featureFlags: string[],
   dataLoader: DataLoaderWorker
 ) => {
   if (featureFlags.includes('noAISummary') || !team) return false
-  const {qualAIMeetingsCount, tier, orgId} = team
+  const {orgId} = team
   const organization = await dataLoader.get('organizations').load(orgId)
   if (organization.featureFlags?.includes('noAISummary')) return false
+  return true
+}
+
+const canAccessAISummary = async (
```

Comment:
-1 this business logic is difficult to reason through.

I see `canAccessAISummary`, `isAISummaryAllowed` and then in another file i see 

```ts
const isAISummaryAccessible =
    (await isAISummaryAllowed(team, facilitator.featureFlags, dataLoader)) &&
    organization.featureFlags?.includes('standupAISummary')
```

can we reduce this to a single function that handles permissions for all `actor, action, resource` cases? Then when we start dropping feature flags it'll be as easy as deleting 1 line.

Reply:
Refactored this a bit